dither/dither.go

```diff
@@ -0,0 +1,56 @@
+package dither
+
+import (
+	"math/rand"
+	"time"
+)
+
+// Dither keeps track of an interval in seconds
+type Dither struct {
+	nextTime int64
+	interval []int64
+}
+
+func (d *Dither) advance() {
+	// use the current interval in the list
+	d.nextTime = time.Now().Add(time.Duration(d.interval[0]) * time.Second).Unix()
+	// if there are more than one remaining interval advance them
+	if len(d.interval) > 1 {
+		d.interval = d.interval[1:]
+	}
+}
+
+// Now returns a boolean indicating if the current time is greater than or equal
+// to the "next" time and the number of seconds until the "next" time
+func (d *Dither) Now() (bool, int64) {
```

Comment:
I'm not sure how you are planning on making use of this, but it seems like it would be way more natural to Go to have this thing act like `time.Ticker` where you select on a channel.

Reply:
Yeah, i think as i asked in my review, a description of the usefulness of this package. Not sure i really get it.